client/data/reader/use-feed-recommendations-query/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { useEffect, useMemo } from 'react';
+import { useDispatch, useSelector } from 'calypso/state';
+import { requestUserRecommendedBlogs } from 'calypso/state/reader/lists/actions';
+import {
+	getUserRecommendedBlogs,
+	hasRequestedUserRecommendedBlogs,
+	isRequestingUserRecommendedBlogs,
+} from 'calypso/state/reader/lists/selectors';
+
+interface QueryOptions {
+	enabled?: boolean;
+}
+
+interface APIFeedRecommendation {
+	meta: {
+		data: {
+			site?: {
+				name: string;
+				feed_URL: string;
+				ID: string;
+				icon: {
+					img?: string;
+					ico?: string;
+				};
+				description: string;
+			};
+			feed?: {
+				image: string;
+				name: string;
+				feed_URL: string;
+				blog_ID: string;
+			};
+		};
+	};
+	feed_ID: string;
+}
+
+export interface FeedRecommendation {
```

Comment:
I am not sure if it is the best name

Reply:
What about `RecommendedBlog` to match the existing terminology?